assessment.rb

```diff
@@ -0,0 +1,7 @@
+1. Explain why Rails exists and what it's used for.
+1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+1. Describe the flow of a PATCH request as it goes through the app.
+
+1. Rails is a way of communicating between the server and the client.  It allows for the transmittion of data stored in the server to be transmitted to the client for rendering, manipulating, reading and other functions.
```

Comment:
> Rails is a way of communicating between the server and the client. 

I don't think this is an accurate depiction.

>It allows for the transmittion of data stored in the server to be transmitted to the client for rendering, manipulating, reading and other functions.

This is true! But it's also true of any back-end. What is Rails? And why do we use it?